prefect_dbt/cli/configs/snowflake.py

```diff
@@ -0,0 +1,24 @@
+"""Module containing models for Snowflake configs"""
+from prefect_dbt.cli.configs.base import MissingExtrasRequireError, TargetConfigs
+
+try:
+    from prefect_snowflake.credentials import SnowflakeCredentials
+except ModuleNotFoundError as e:
+    raise MissingExtrasRequireError("Snowflake") from e
+
+
+class SnowflakeTargetConfigs(TargetConfigs):
+    """
+    Target configs contain credentials and
+    settings, specific to Snowflake.
+    To find valid keys, head to the [Snowflake Profile](
+    https://docs.getdbt.com/reference/warehouse-profiles/snowflake-profile)
+    page.
+
+    Args:
+        credentials: The credentials to use to authenticate; if there are
+            overlapping keys between credentials and TargetConfigs,
+            e.g. schema, credentials takes precedence.
+    """
+
+    credentials: SnowflakeCredentials
```

Comment:
Does this class need to override `get_configs` to create the config in the format that dbt is expecting?

Reply:
No need to override I think.

Reply:
https://github.com/PrefectHQ/prefect-dbt/pull/22/files#diff-36b352fcdfeb31d70883c5795dcbe4b86d88e7e5edd75260405e09413c082ff5R24-R31

Reply:
Can you post the output from calling `.get_configs()` on an instantiation of `SnowflakeTargetConfigs`?

Reply:
```
from prefect_dbt.cli.configs import SnowflakeTargetConfigs
from prefect_snowflake.credentials import SnowflakeCredentials
snowflake_credentials = SnowflakeCredentials(account="account", user="user", password="password")
SnowflakeTargetConfigs(credentials=snowflake_credentials, type="snowflake", schema="schema")
```
yields
```
SnowflakeTargetConfigs(extras=None, type='snowflake', schema_='schema', threads=4, credentials=SnowflakeCredentials(account='account', user='user', password=SecretStr('**********'), database=None, warehouse=None, private_key=None, authenticator=None, token=None, schema_=None, role=None, autocommit=None, connect_params={'account': 'account', 'user': 'user', 'password': SecretStr('**********'), 'application': 'Prefect_Snowflake_Collection'}))
```

Reply:
Okay yea, I think I need to remove connect_params...